python/dgl/distributed/partition.py

```diff
@@ -1561,6 +1616,8 @@ def dgl_partition_to_graphbolt(
     store_inner_edge=False,
     graph_formats=None,
     n_jobs=1,
+    parts=None,
```

Comment:
this is a public API, is it possible to do not change the interface?

Reply:
I added a new function for graphbolt and keep the API unchanged.